cmd/minishift/cmd/addon/addon_apply.go

```diff
@@ -77,34 +77,34 @@ func runApplyAddon(cmd *cobra.Command, args []string) {
 
 	ip, err := host.Driver.GetIP()
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Error getting IP: %s", err.Error()))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error getting the IP address: %s", err.Error()))
 	}
 
 	routingSuffix := determineRoutingSuffix(host.Driver)
 	sshCommander := provision.GenericSSHCommander{Driver: host.Driver}
 	ocRunner, err := oc.NewOcRunner(minishiftConfig.InstanceConfig.OcPath, constants.KubeConfigPath)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Error applying addon: %s", err.Error()))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error applying the add-on: %s", err.Error()))
 	}
 
 	for i := range args {
 		addonName := args[i]
 		addon := addOnManager.Get(addonName)
 		addonContext, err := clusterup.GetExecutionContext(ip, routingSuffix, ocRunner, sshCommander)
 		if err != nil {
-			atexit.ExitWithMessage(1, fmt.Sprint("Error executing addon commands: ", err))
+			atexit.ExitWithMessage(1, fmt.Sprint("Error running add-on commands: ", err))
```

Comment:
> The reason I had written this because I wanted an easier message to understand. Do you think we need to change the current message? or we can just leave it as it is.

I think it needs to change. Whether we mention execution context or whether we got with the more generic "Error applying add-on %s: %v" I don't mind.